packages/devtools_app/lib/src/memory/memory_snapshot_models.dart

```diff
@@ -292,6 +295,30 @@ class AnalysisField extends TreeNode<AnalysisField> {
   final String value;
 }
 
+/// Snapshot being analyzed.
+class SnapshotReference extends Reference {
+  SnapshotReference(this.snapshot)
+      : super.snapshot(
+          null,
+          snapshot != null
+              ? 'Snapshot ${_formatter.format(snapshot.collectedTimestamp)}'
+              : '',
+/*
+          onExpand: (Reference reference) {
+            assert(reference.isAnalysis);
+            print("TBD");
+          },
+          onLeaf: (ObjectReference reference) {
+            print("Clicking on analysis node");
+          },
+*/
+        );
+
+  static final _formatter = DateFormat('MMM dd HH:mm:ss');
```

Comment:
remove this formatter?

Reply:
Yep should have use my MemoryController formattedTimestamp thanks.